spec/models/counters_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'spec_helper'
+require 'spec/support/persistence'
+require 'routemaster/models/counters'
+
+describe Routemaster::Models::Counters do
+  subject { described_class.instance }
+
+  after { subject.finalize }
+
+  describe '#incr' do
+    it 'passes' do
+      expect { subject.incr(:foo) }.not_to raise_error
+    end
+
+    it 'allows tags' do
+      expect { subject.incr(:foo, bar: 2) }.not_to raise_error
+    end
```

Comment:
`not_to raise_error` is quite a weak test as just a blank method body would pass. Is it possible to write a stronger test here?